src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -4631,7 +4631,7 @@ public override bool PreVisit(CallNode node)
                 var overloads = LookupFunctions(funcNamespace, node.Head.Name.Value);
                 if (!overloads.Any())
                 {
-                    _txb.ErrorContainer.Error(node, TexlStrings.ErrUnknownFunction);
+                    _txb.ErrorContainer.Error(node, TexlStrings.ErrUnknownFunction, node.Head.Name.Value);
```

Comment:
>node.Head.Name.Value [](http://example.com/codeflow?start=84&length=20)

If the funcNamespace is not global or empty then you may want to include it here.